l10n_th_fields/models/partner.py

```diff
@@ -61,11 +61,8 @@ def _check_partner_name(self):
     @api.constrains('vat', 'taxbranch')
     def _check_vat_taxbranch_unique(self):
         if self.vat or self.taxbranch:
-            count = self.search_count(
-                ['|', ('parent_id', '=', False),
-                 ('is_company', '=', True),
-                 ('vat', '=', self.vat),
-                 ('taxbranch', '=', self.taxbranch)])
+            count = self.search_count([('vat', '=', self.vat),
+                                       ('taxbranch', '=', self.taxbranch)])
```

Comment:
@yostashiro all the l10n_th are localization modules are maintained in separated repository. And I also keep adding more features, so it will be dangerous if it is changed here.
Is the above are all you changed for l10n_th? I can merge to my main branch. Can you tell me what is it for?
Note: I was having problem when I pull the lastest cft code last week. Got warning saving a customer, even it has unique vat + taxbranch.